frontend/workflows/projectSelector/package.json

```diff
@@ -25,7 +25,8 @@
     "@material-ui/icons": "^4.11.2",
     "lodash": "^4.17.21",
     "react": "^17.0.2",
-    "react-dom": "^17.0.2"
+    "react-dom": "^17.0.2",
+    "react-hook-form": "^6.11.0"
```

Comment:
can you update this to ^7.25.3 to match the housekeeping PR that just landed on main?

Reply:
Yup, will do